src/EA.Weee.Web/Areas/Producer/Mappings/ToViewModel/ServiceOfNoticeMap.cs

```diff
@@ -23,7 +23,7 @@ public ServiceOfNoticeViewModel Map(SmallProducerSubmissionMapperData source)
             {
                 DirectRegistrantId = submissionData.DirectRegistrantId,
                 OrganisationId = submissionData.OrganisationData.Id,
-                Address = source.UseMasterVersion ? MapAddress(submissionData.ServiceOfNoticeData) : MapAddress(submissionData.CurrentSubmission.ServiceOfNoticeData),
```

Comment:
@stevelee-civica this line Address = source.UseMasterVersion ? MapAddress(submissionData.ServiceOfNoticeData) : MapAddress(submissionData.CurrentSubmission.ServiceOfNoticeData), looks correct to me